modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -157,6 +168,36 @@ export interface ListWalletOptions extends PaginationOptions {
   allTokens?: boolean;
 }
 
+export interface WalletShare {
+  id: string;
+  coin: string;
+  wallet: string;
+  walletLabel: string | undefined;
+  fromUser: string;
+  toUser: string;
+  permissions: string[];
+  state: string | undefined;
+  enterprise?: string;
+  keychainOverrideRequired?: boolean;
+  isUMSInitiated?: boolean;
+  keychain?: {
+    pub: string;
+    encryptedPrv: string;
+    fromPubKey: string;
+    toPubKey: string;
+    path: string;
+  };
+}
+
+export interface WalletShares {
+  incoming: WalletShare[];
+  outgoing: WalletShare[];
+}
```

Comment:
Please give more context here what 'incoming` and `outgoing` means. This type seems kinda vague